File: eTierMeeting_v2/SPA/src/app/_core/_dtos/search-machine-params.ts
export interface SearchMachineParams {
  machineId: string;
  pdcId: number | null;
  buildingCode: string;
  buildingId: number;
  cellCode: string;
  category: string;
  sort: string;
  isPaging: boolean;
  positionCode: string;
}

File: eTierMeeting_v2/API/Helpers/Params/SearchMachineParams.cs
namespace Machine_API.Helpers.Params
{
    public class SearchMachineParams
    {
        public string MachineId { get; set; }
        public int? PdcId { get; set; }
        public int? BuildingCode { get; set; }
        public int BuildingId { get; set; }
        public string CellCode { get; set; }
        public string PositionCode { get; set; }
        public string Category { get; set; }
        public string Sort { get; set; }
        public bool IsPaging { get; set; } = true;
    }
}